modules/benchmark/R/start.bm.ensemble.R

```diff
@@ -1,21 +1,44 @@
 ##-------------------------------------------------------------------------------------------------#
-##' Start Benchmark Run
+##' Start Benchmark Ensemble
 ##'  
-##' @name start.bm.run
-##' @title Start benchmark run
-##' @param benchmark_reference_run.id,
-##' @param machine.id
-##' @export 
+##' @name start.bm.ensemble
+##' @title Start benchmark ensemble
+##' @param benchmark_reference_run.id
+##' @param con database connection
+##' @export start.bm.ensemble
 ##' 
 ##' @author Betsy Cowdery 
 
-start.bm.run <- function(BRR.id, database, machine.id){ #- leverage workflow.R
+start.bm.ensemble <- function(BRR, con){ 
+  
+    old.settings <- file.path(BRR$settings, "pecan.xml")
+    new.settings <<- sprintf("%s/%s.pecan.xml",BRR$settings, basename(BRR$settings))
+    clean.settings(old.settings,new.settings)
```

Comment:
this needed to have been in create.BRR. Here you need to be taking that cleaned settings, making any machine-specific updates that read.settings won't handle, and writing it out.  Indeed, to be able to do that a more generalized create.BRR will (eventually) need to know how to re-generalize machine-specific values back to their general values (e.g. input.ids instead of paths) and possibly insert tags that would get replaced here.